packages/next-acms/src/test-api-connection.ts

```diff
@@ -0,0 +1,24 @@
+import { DrupalClient } from 'next-drupal';
+
+export async function testApiConnection(
```

Comment:
This would be a good example of something that could use unit tests. I think we should work on that next.

Reply:
@harumijang Do you have thoughts on the name? I feel like `testApiConnection` is not great. I'm wondering if `testApiCompatibility` would be better?

Reply:
What about `testApiIntegration`?

Reply:
I am ok with `testApiCompatibility` too